src/node_watchdog.h

```diff
@@ -91,6 +91,7 @@ class SigintWatchdogHelper {
   static void* RunSigintWatchdog(void* arg);
   static void HandleSignal(int signum);
 #else
+  bool watchdog_disabled_;
```

Comment:
It would be nicer, but with `watchdog_disabled_` it is easier to tell if `SigintWatchdogHelper::Start()` is called for the first time and we have to register the handler or it is called after `::Stop()` and we just have to flip the flag.